docs/developer/running-saleor/task-queue.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: Task Queue
+sidebar_label: Task Queue
+---
+
+Some of the Saleor’s functionality depends on a task queue system to manage asynchronous and periodic tasks. We recommend using Celery with Redis as the message broker.
+For more information, please visit [official Celery documentation](https://docs.celeryq.dev/en/stable/index.html). 
+
+
+## Periodic tasks
+
+Celery Beat is a scheduler that triggers background tasks at regular intervals. 
+Some of the Saleor's features can't be used without proper task scheduler setup. In order to start Celery Beat server:
+
+1. Specify `CELERY_BROKER_URL` environmental variable which points to task broker (we recommend using Redis as a broker database), i.e.:
+
+```shell
+set CELERY_BROKER_URL="redis://localhost:6379/0"
```

Comment:
Maybe worth adding that `set CELERY_BROKER_URL="redis://localhost:6379/0"` is an example of setting the redis URL. 
Maybe worth mentioning, that this can be different depending on the setup/configuration of the broker? Current steps suggests mi flow jak `ctrl+c & ctrl+v` in the shell, but this steps would requiring additional action to find the broker address.

Reply:
Do you mean something like:
set CELERY_BROKER_URL=<broker url>